packages/feedback-service/src/resolver.ts

```diff
@@ -153,13 +153,12 @@ export const FeedbackResolver = {
         return Feedback.findById( response._id )
           .then( fb => {
             if ( fb ) {
-
               const issue = {
                 'fields': {
                   'project': {
                     'key': process.env.PROJECT_KEY
                   },
-                  'summary': fb.title,
+                  'summary': ( fb.feedbackType === 'Bug' ? `${ fb.title }` : `Feedback`),
```

Comment:
Can you change the default title to "Feedback for ${spa_name}"

```suggestion
                  'summary': ( fb.feedbackType === 'Bug' ? `${ fb.title }` : `Feedback`),
```